LayoutTests/imported/w3c/web-platform-tests/css/css-transitions/transition-important.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<title>CSS Transitions Test: !important property</title>
+<link rel="help" href="https://drafts.csswg.org/css-cascade-5/#cascade-sort">
+<script src="/resources/testharness.js" type="text/javascript"></script>
+<script src="/resources/testharnessreport.js" type="text/javascript"></script>
+<style>
+#target {
+    width: 200px;
+    height: 200px;
+    background-color: green;
+    transition: background-color 1s;
+}
+.red {
+    background-color: red !important;
+}
+</style>
+<div id="target"></div>
+<script>
+test(t => {
+    document.body.offsetLeft;
+    target.className = "red";
+    assert_equals(getComputedStyle(target).backgroundColor, "rgb(0, 128, 0)");
```

Comment:
While it's functionally equivalent, I think it would be better form to query the computed style here instead of getting `offsetLeft` to ensure a style update prior to apply transitions.

```suggestion
    const cs = getComputedStyle(target);
    target.className = "red";
    assert_equals(cs.backgroundColor, "rgb(0, 128, 0)");
```